workflows/scnanoseq.nf

```diff
@@ -1,15 +1,109 @@
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    IMPORT MODULES / SUBWORKFLOWS / FUNCTIONS
+    PARAMETER PRESETS
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-include { FASTQC                 } from '../modules/nf-core/fastqc/main'
-include { MULTIQC                } from '../modules/nf-core/multiqc/main'
-include { paramsSummaryMap       } from 'plugin/nf-validation'
-include { paramsSummaryMultiqc   } from '../subworkflows/nf-core/utils_nfcore_pipeline'
-include { softwareVersionsToYAML } from '../subworkflows/nf-core/utils_nfcore_pipeline'
-include { methodsDescriptionText } from '../subworkflows/local/utils_nfcore_scnanoseq_pipeline'
+
+if (params.barcode_format.equals("10X_3v3")) {
+    blaze_whitelist = file("$baseDir/assets/whitelist/3M-february-2018.zip")
+}
+else{
+    blaze_whitelist = file("$baseDir/assets/whitelist/737K-august-2016.txt.zip")
```

Comment:
shouldn't then be `10X_5v2` the default value for `barcode_format`?

Reply:
Yeah it does read like that, I think it may make more sense to turn the second `else` into an `else if (params.barcode_format.equals("10X_5v2")` for readability, since this param can only be one of those two values anyway so it makes it more clear

Reply:
Tried to address this with commit c082e20